nextcord/ui/item.py

```diff
@@ -100,8 +91,28 @@ def row(self, value: Optional[int]) -> None:
     def width(self) -> int:
         return 1
 
+
+class ViewItem(Item, Generic[ViewT_co]):
+    """Represents the View UI item that all UI components supporting :class:`View` inherit from.
+
+    The current UI items supported for View are:
+
+    - :class:`nextcord.ui.Button`
+    - :class:`nextcord.ui.StringSelect`
+    - :class:`nextcord.ui.UserSelect`
+    - :class:`nextcord.ui.ChannelSelect`
+    - :class:`nextcord.ui.RoleSelect`
+    - :class:`nextcord.ui.MentionableSelect`
+
+    .. versionadded:: 2.0
```

Comment:
This doesnt seem right. Im not sure if it should be versionadded & versionchanged or simply version added (given these are technically new user facing imports)